pages/your_tasks/data_brokering.md

```diff
@@ -0,0 +1,89 @@
+* Identify who is responsible for the data before, during and after delivery, such as the data controller/processor (according to GDPR) and/or intellectual property owner/licensee relationships between the provider and recipient
+* Ensure that you will be able to establish contracts/agreements that cover the data and processing that you will handle, such as considerations for [data protection](https://rdmkit.elixir-europe.org/data_protection.html), [licensing](https://rdmkit.elixir-europe.org/licensing.html) and [compliance](https://rdmkit.elixir-europe.org/compliance_monitoring.html).
+* Estimate and secure the resources required to keep your commitment, such as staff with time and necessary skills, accounts, compute, storage and software
+* Refer to the sections below for considerations related to collecting data from data providers and delivering data to public data repositories.
+
+
+### Solutions
+
+The solutions that you adopt will vary depending on the agreements you have negotiated with data providers and/or recipients. The following are examples of general solutions that would help you comply with regulations and implement good data management practices.
+* [Data management plan](https://rdmkit.elixir-europe.org/data_management_plan.html) – Many questions that you would answer while writing a data management plan can be relevant to answer when you specify the terms of service for your brokering service, such as data storage, data standards, legal and ethical, etc. 
+* [Data protection](https://rdmkit.elixir-europe.org/data_protection.html) – If you are working with data concerning people in the EU, you should make sure to comply with both national and international regulations for data protection.
+* [Apply for brokering permissions](https://ena-docs.readthedocs.io/en/latest/faq/data_brokering.html) e.g. [ENA support/helpdesk](https://www.ebi.ac.uk/ena/browser/support) 
+
+
+## Collecting and processing the metadata and data
+### Description
+Data brokering involves collecting data from various data providers (metadata and other data e.g. sequencing files), standardising and curating the data if needed, and then preparing the data for re-sharing e.g. in public international repositories. On the brokering platform, it is recommended that data be stored in a structured manner (e.g. in a relational database), using as much as possible controlled vocabularies and ontologies where they exist.
+
+### Considerations
+* [Data collection](https://rdmkit.elixir-europe.org/collecting) should be carefully prepared, notably to define the data model, the metadata and data that are needed for the envisioned applications, assess which fields should be compulsory or optional, follow controlled vocabularies or ontologies, and identify the nature of data (personal, sensitive data) and thereby the required level of security or data treatment (e.g. pseudonymised or anonymised data, ethical consent…).
```

Comment:
```suggestion
* [Data collection](collecting) should be carefully prepared, notably to define the data model, the metadata and data that are needed for the envisioned applications, assess which fields should be compulsory or optional, follow controlled vocabularies or ontologies, and identify the nature of data (personal, sensitive data) and thereby the required level of security or data treatment (e.g. pseudonymised or anonymised data, ethical consent…).
```

Reply:
are 'and' and ''or' highlighted in red means they will be excluded in final changes?